03-task-manager/starter/app.js

```diff
@@ -1 +1,33 @@
-console.log('Task Manager App')
+const express = require('express');
+const app = express();
+const tasks = require('./routes/tasks');
+const connectDB = require('./db/connect');
+require('dotenv').config();
+//const MONGO_URI = process.env.MONGO_URI;
+//middleware
+app.use(express.json());
+
+//routes
+app.get('/hello', (req, res) => {
+  res.status(200).send('Task Manager App');
+});
+
+app.use('/api/v1/tasks', tasks);
+//app.get('/api/v1/tasks') - get all the tasks
+// app.post('/api/v1/tasks') - create a new task
+// app.get('/api/v1/tasks/:id') - get single task
+// app.patch('/api/v1/tasks/:id') - update task
+//app.delete('/api/v1/tasks/:id') - delete task
+
+const port = 5000;
+
+const start = async () => {
+  try {
+    await connectDB(process.env.MONGO_URI);
+    app.listen(port, console.log(`Server is listening on port ${port}`));
```

Comment:
You probably want to wrap console.log(`Server is listening on port ${port}`) in a callback function. As is, the console.log statement will be invoked immediately rather than after the app actually starts. See here for an example of how it's supposed to work: https://expressjs.com/en/starter/hello-world.html